lava-v2-jobs-from-api.py

```diff
@@ -157,32 +184,59 @@ def main(args):
                                         if 'BIG_ENDIAN' in defconfig and plan == 'boot-be':
                                             endian = 'big'
                                         initrd_arch = arch
-                                        if arch not in INITRD_URLS.keys():
+                                        if arch not in ARCHS:
                                             if arch == 'arm64' and endian == 'big':
                                                 initrd_arch = 'arm64be'
                                             if arch == 'arm':
                                                 if endian == 'big':
                                                     initrd_arch = 'armeb'
                                                 else:
                                                     initrd_arch = 'armel'
-                                        initrd_url = INITRD_URLS[initrd_arch]
-                                        if 'nfs' in plan:
-                                            nfsrootfs_url = NFSROOTFS_URLS[initrd_arch]
-                                        else:
-                                            nfsrootfs_url = None
+                                        initrd_url = INITRD_URL.format(initrd_arch)
+                                        nfsrootfs_url = NFSROOTFS_URL.format(initrd_arch) if 'nfs' in plan else None
```

Comment:
Well, I personally find it better to have one line to define the value of a variable.  Can revert that if it's breaking the coding style, it's part of the change to refactor the list of URLs.